src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -127,4 +112,17 @@ OptionRowLHNData.displayName = 'OptionRowLHNData';
  * Thats also why the React.memo is used on the outer component here, as we just
  * use it to prevent re-renders from parent re-renders.
  */
-export default React.memo(OptionRowLHNData);
+export default withOnyx({
+    policy: {
+        key: ({fullReport}) => `${ONYXKEYS.COLLECTION.POLICY}${fullReport.policyID}`,
+        initialValue: {},
+    },
+    comment: {
+        key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`,
+        initialValue: '',
+    },
+    transaction: {
+        key: ({transactionID}) => `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`,
+        initialValue: {},
+    },
+})(React.memo(OptionRowLHNData));
```

Comment:
I just want to double-check it does not create any overhead when scrolling the list with large amount of items - we removed this HOC from this component before Flashlist migration, as it was constantly calling `Onyx.connect` while scrolling the list, making it really unperformant. The best option to test it would be account with as much reports as possible (e.g. Edu's newdotperf account with 15k records)